Library/Formula/openjpeg.rb

```diff
@@ -22,7 +22,13 @@ class Openjpeg < Formula
   depends_on "libpng"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    system "cmake", ".", *args
+    system "make", "install"
```

Comment:
:( There is no acceptable way to get both the shared and static libraries in the bottle, as for most libraries?